ccd/deployment.yaml

```diff
@@ -0,0 +1,825 @@
+---
```

Comment:
might be worth pulling out each app to its own file?

Reply:
if we use Helm I will do that.  For now though if it's all in one file teams can copy/paste into their `deployment.template.yaml`